view_post.php

```diff
@@ -0,0 +1,62 @@
+<html>
+
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+
+	<body>
+
+<?php
+
+$sorter= array(
+	'Blog1'=>array(
+		'<h1 class="title">Halfway Done</h1>',
```

Comment:
The `<h1>` is common for all of these headers, so why not leave it out (just make this "Halfway Done" and echo the boilerplate html below when you're echoing the dynamic content.